Makefile

```diff
@@ -81,8 +99,8 @@ update: _update_yamls _update_crds _reset_yamls
 
 ### uninstall: remove namespace and all CRDs from cluster
 uninstall:
-	# It's safer to delete all workspaces before deleting the controller; otherwise we could
-	# leave workspaces in a hanging state if we add finalizers.
+# It's safer to delete all workspaces before deleting the controller; otherwise we could
+# leave workspaces in a hanging state if we add finalizers.
 ifneq ($(shell command -v kubectl),)
```

Comment:
it does not work for me for some reasons I don't understand because of limited knowledge of makefiles
![Screenshot_20200403_124352](https://user-images.githubusercontent.com/5887312/78347298-f0047180-75a8-11ea-8a88-17df4fca4a16.png)

Reply:
Figured it out, that was subtle: https://stackoverflow.com/questions/5618615/check-if-a-program-exists-from-a-makefile#comment76110496_34756868